chat-plugins/info.js

```diff
@@ -264,50 +264,55 @@ exports.commands = {
 	ipsearchall: 'ipsearch',
 	hostsearch: 'ipsearch',
 	ipsearch: function (target, room, user, connection, cmd) {
-		if (!target.trim()) return this.parse('/help ipsearch');
+		if (!target.trim()) return this.parse(`/help ipsearch`);
 		if (!this.can('rangeban')) return;
-		let results = [];
 
+		target = this.splitTargetText(target).trim();
+		let tarRoom = target.length ? Rooms(target.trim()) : null;
+		if (!tarRoom && tarRoom !== null) return this.errorReply(`The room "${target}" does not exist.`);
+		let results = [];
 		let isAll = (cmd === 'ipsearchall');
 
-		if (/[a-z]/.test(target)) {
+		if (/[a-z]/.test(this.targetUsername)) {
```

Comment:
Should probably do `const targetIp = this.targetUsername;` so future references are clearer.

Reply:
Using `let` instead because its modified when searching an ip range
``targetIp = targetIp.slice(0, -1);``